activiti-api-impl/activiti-api-task-runtime-impl/src/main/java/org/activiti/runtime/api/impl/TaskAdminRuntimeImpl.java

```diff
@@ -135,46 +138,91 @@ public Task release(ReleaseTaskPayload releaseTaskPayload) {
         taskService.unclaim(releaseTaskPayload.getTaskId());
         return task(releaseTaskPayload.getTaskId());
     }
-
+    
     @Override
-    public void addCandidateGroups(CandidateGroupsPayload arg0) {
-        // TODO Auto-generated method stub
+    public Task assign(AssignTaskPayload assignTaskPayload) {
+        taskService.claim(assignTaskPayload.getTaskId(),
```

Comment:
@CTI777 we need a test for when the task does have an assignee and for when the task doesn't have an assignee